dm/config/subtask.go

```diff
@@ -268,8 +269,11 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 		c.ShardMode = ShardPessimistic // use the pessimistic mode as default for back compatible.
 	}
 
-	if c.OnlineDDLScheme != "" && c.OnlineDDLScheme != PT && c.OnlineDDLScheme != GHOST {
+	if !c.OnlineDDL && c.OnlineDDLScheme != "" && c.OnlineDDLScheme != PT && c.OnlineDDLScheme != GHOST {
 		return terror.ErrConfigOnlineSchemeNotSupport.Generate(c.OnlineDDLScheme)
+	} else if c.OnlineDDLScheme == PT || c.OnlineDDLScheme == GHOST {
+		c.OnlineDDL = true
+		log.L().Warn("'online-ddl-scheme' will be deprecated soon. Recommend that use online-ddl instead of online-ddl-scheme.")
```

Comment:
please remeber to update doc in https://github.com/pingcap/docs-dm

Reply:
Shall I also change the file name `feature-online-ddl-scheme.md` to `feature-online-ddl.md`? 
https://github.com/pingcap/docs-dm/blob/master/zh/feature-online-ddl-scheme.md

Reply:
i think it's ok to rename file

Reply:
PTAL. https://github.com/pingcap/docs-dm/pull/734